<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QWidget" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>184</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;
color: #34421E;</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>155</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#E60936;
border-radius: 5px;
color: #fff;
font-size: 14px;
</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>49</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: #040F3D;
font-size: 12px;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-size: 12px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font-size: 12px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="portLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius: 8px;
border: 1px solid #ccc;
background-color: #DEC79B;
}

QLineEdit:focus{
border: 2px solid #39c;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ipLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius: 8px;
border: 1px solid #ccc;
background-color: #DEC79B;
}

QLineEdit:focus{
border: 2px solid #39c;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius: 8px;
border: 1px solid #ccc;
background-color: #DEC79B;
}

QLineEdit:focus{
border: 2px solid #39c;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="connectLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
